src/components/m-dropdown/js/drop-down.js

```diff
@@ -11,6 +11,7 @@ class DropDown extends UiEvents {
     isOpenClass: 'is-dropdown-open',
     isAnimatingClass: 'is-dropdown-animating',
     selectClass: 'js-dropdown__content',
+    selectedItemSelector: 'option[selected=selected]',
```

Comment:
will this work in all the browser? maybe better `option[selected="selected"]`

Reply:
This will  be removed, it is part of the old implementation that I forgot to clean up and it is not used atm.